src/lib/plugins/cpu.js

```diff
@@ -98,33 +98,36 @@ export function cpuFactory( React, colors ) {
       return averageCpuUsage
     }
 
-    getColor( cpuAverage ) {
-      const CPU_USAGE_COLORS = {
-        HIGH: colors.lightRed,
-        MODERATE: colors.lightYellow,
-        LOW: colors.lightGreen
-      }
+    getColor(cpuAverage) {
+      const colors = this.props.options.colors
 
       if ( cpuAverage < 50 ) {
```

Comment:
Switch statements are definitely a good practice for most other cases, thanks for the review 👍